.circleci/config.yml

```diff
@@ -119,6 +119,7 @@ jobs:
     environment:
       CC_TEST_REPORTER_ID: faecd27e9aed532634b3f4d3e251542d7de9457cfca96a94208a63270ef9b42e
       COVERAGE: true
+      RAILS_ENV: test
```

Comment:
for running locally, are we going to need to update any of our scripts for things in our test suite that assume `Rails.env.test?`

Reply:
I don't think so. CircleCI is having some puzzling issues with the database for this set of changes that I'm still trying to figure out